docs/publish-module/publish-information/check-release-status.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Check Release Status
+description: Learn how to check and update statuses in the Publish module of Appcircle
+tags: [publish, publish module, app information, app store connect, review]
+sidebar_position: 2
+---
+
+# Check Release Status
+
+Appcircle performs status checks for your published applications. This status gives you information about the status of the relevant version on the market. You can find detailed information about Appcircle's status feature in the documentation [**here**](/publish-module/binary-management#store-status).
+
+Appcircle updates a published app for status updates **every 30 minutes** if the app is [**Marked as RC**](/publish-module/publish-information/marking-release-candidates). However, the user can also do this manually.
+
+:::caution Check Release Status Manually
+
+This feature is currently only available for **iOS profiles**. It will be available for Android profiles as soon as possible.
+
+:::
+
+With the Check Release Status function, you can instantly update the status information of the version marked as Release Candidate.
+
+:::info
+
+Make sure that one of the versions is [**Marked as Release Candidate**](/publish-module/publish-information/marking-release-candidates) so that you can do a Status Check.
+
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE4007-releaseStatus.png' />
+
+
+:::danger Check Release Status
+
+In order not to exceed **API request limits**, and not to cause any other problems, manual status checking is limited to one request **every 10 minutes**.
```

Comment:
```suggestion
To avoid exceeding **API request limits** and causing other issues, manual status checks are limited to one request **every 10 minutes**.
```